environment.yaml

```diff
@@ -0,0 +1,58 @@
+name: pyinknife2-env
+channels:
+  - conda-forge
+dependencies:
```

Comment:
there's a double `- pip:` here. This section should be something like

```
dependencies:
    - python==3.7
    - pip
    - pip:
        - pakage1=...
 ```

Reply:
ok!